common.blocks/toolbar/toolbar.deps.js

```diff
@@ -2,9 +2,8 @@
     mustDeps: ['i-bem-dom'],
     shouldDeps: [
         {
-            elems: ['user', 'form']
```

Comment:
I'm not quite sure if it's really something reasonable but some time ago we moved `user` from `header` to `toolbar`. Let's discuss it offline today?